rfcs/xxxx-rfc-process/text.md

```diff
@@ -0,0 +1,60 @@
+# RFC Process <img style="float: right;" src="./imgs/monkey_mascot.jpg" width="200">
+
+**Authors:** [Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+**Approvers:** [Timmy Huang](mailto:thuang@chanzuckerberg.com), [Marcus Kinsella](mkinsella@chanzuckerberg.com), [Brian Raymor](mailto:braymor@chanzuckerberg.com)
+
+**Reviewers:** [Madison Dunitz](madison.dunitz@chanzuckerberg.com), [Brian McCandless](bmccandless@chanzuckerberg.com), [Trent Smith](trent.smith@chanzuckerberg.com), [Colin Megill](colin.megill@chanzuckerberg.com), [Seve Badajoz](sbadajoz@chanzuckerberg.com)
+
+## tl;dr 
+
+This RFC contains the details about the RFC process for all cellxgene related designs.
+
+
+## Background
+
+Previously the "RFC" (a.k.a. "design doc review") process was executed on Google Docs. There were two problems
+
+1. Updates were quite tricky to make. Do you update the doc directly? Do you have a changelog? Which version is latest? Do you need to apply the change in the changelog in the body of the doc? 
+2. Designs decisions were not easily accessible by our external community. How does the external community, with whom we have a close relationship, keep track of our decisions and progress if the designs are hidden in a private Google Drive?
+
+In order to alleviate these questions and concerns, we have moved to Github to keep track of our RFCs moving forward.
+
```

Comment:
I was expecting to see a section on **When is an RFC required?** but perhaps that's part of a separate software practice discussion.

Reply:
I added a section about it which is kind of strawman but part of it will be also a practice/cultural discussion.